### PR TITLE
Update version.py

### PR DESCRIPTION
**Pull request by seeker25**

### CODE REVIEW

File: queue_services/payment-reconciliations/src/reconciliations/cgi_reconciliations.py [hunk 1]
```diff
@@ -96,7 +96,9 @@ async def _update_feedback(msg: Dict[str, any]):  # pylint:disable=too-many-loca
     content = file.data.decode('utf-8-sig')
     group_batches: List[str] = _group_batches(content)
     has_errors = await _process_ejv_feedback(group_batches['EJV'])
-    has_errors = await _process_ap_feedback(group_batches['AP']) or has_errors
+
+    if not APP_CONFIG.DISABLE_AP_FEEDBACK:
```
Comment (seeker25) [inline queue_services/payment-reconciliations/src/reconciliations/cgi_reconciliations.py:100]: Already in main

File: queue_services/payment-reconciliations/src/reconciliations/config.py [hunk 2]
```diff
@@ -117,6 +117,8 @@ class _Config():  # pylint: disable=too-few-public-methods
 
     # Disable EJV Error Email
     DISABLE_EJV_ERROR_EMAIL = os.getenv('DISABLE_EJV_ERROR_EMAIL', 'true').lower() == 'true'
+    # Disable AP Feedback processing
+    DISABLE_AP_FEEDBACK = os.getenv('DISABLE_AP_FEEDBACK', 'true').lower() == 'true'
```
Comment (seeker25) [inline queue_services/payment-reconciliations/src/reconciliations/config.py:121]: Already in main

File: queue_services/events-listener/requirements/dev.txt [hunk 3]
```diff
@@ -27,5 +27,5 @@ isort
 
 # docker
 lovely-pytest-docker
-pytest-asyncio
+pytest-asyncio==0.18.3
```
Comment (seeker25) [inline queue_services/events-listener/requirements/dev.txt:30]: Already in main

File: .github/workflows/payment-reconciliations-ci.yml [hunk 4]
```diff
@@ -66,6 +66,7 @@ jobs:
       CFS_CLIENT_ID: "TEST"
       CFS_CLIENT_SECRET: "TEST"
       ACCOUNT_SECRET_KEY: "abcdefg"
+      DISABLE_AP_FEEDBACK: False
```
Comment (seeker25) [inline .github/workflows/payment-reconciliations-ci.yml:69]: Already in main

**Review by shabeeb-aot — APPROVED** [review]

**Pull request merged**